src/server/index.ts

```diff
@@ -15,12 +15,19 @@ export const stopServer = (): void => {
   server?.close()
 }
 
+function symlinkEnvDir() {
+  const repoPath = '/tmp/otomi-values'
+  const envPath = 'env'
+  if (!existsSync(repoPath)) throw new Error(`The git repository does not exist at ${repoPath} path`)
```

Comment:
This is never the case as we designed it to be there. No need for defending, and expecting it to "may be" different.